.azure/pipelines/ci.yml

```diff
@@ -176,8 +176,7 @@ stages:
                 -noBuildDeps
                 $(_BuildArgs)
                 $(_InternalRuntimeDownloadArgs)
-        # Disabled until 3.1.3 is released
-        condition: false
+        condition: ne(variables['Build.Reason'], 'PullRequest')
```

Comment:
FYI this step doesn't have a `condition` on it in 'release/3.1' branch, meaning it will run if previous steps succeed.